.circleci/config.yml

```diff
@@ -158,6 +162,10 @@ jobs:
       # CACHIX_AUTH_TOKEN is manually set in the CircleCI web UI and allows us to push to CACHIX_NAME.
       CACHIX_NAME: "privatestorage-opensource"
 
+      # Let us use features marked "experimental".  For example, most/all of
+      # the `nix <subcommand>` forms.
+      NIX_CONFIG: "experimental-features = nix-command"
```

Comment:
Ah, I guess we use `nix path-info` below, which is why we need this. We could consider using `nix build -v -f path/to/file.nix` for the build as well.